src/main/pages/user-management/organizations.adoc

```diff
@@ -64,77 +81,120 @@ CHE_MAIL_SMTP_AUTH_USERNAME=no-reply@gmail.com
+Members of an organization can not request to leave an organization or sub-organization. You contact the administrator of the organization or the system administrator to request that a member leaves an organization or sub-organization.  
 
 [id="delete-an-organization-or-sub-organization"]
-== Delete an Organization or Sub-Organization
+== Deleting an organization or sub-organization
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+To delete an organization or a sub-organization:
 
-To delete an organization or a sub-organization, click on "Delete" button. This action can’t be reverted and all workspaces created under the organization will be deleted.
+* Click on "Delete" button.
 
-All members of the organization will receive an email notification to inform about organization deletion.
+[IMPORTANT]
+====
+This action can not be reverted and all workspaces created under the organization will be deleted.
+All members of the organization will receive an email notification to inform about the deletion of the organization. Only system adminstrators or administrators of the organization delete an organization or sub-organization. 
+====
 
 [id="organization-resources"]
-== Organization Resources
+== Allocating resources for organizations
 
-Workspaces created in an organization use the resources allocated by the system administrator to that organization. Resources for sub-organizations are taken from the parent organization. Admins can control whether all, or a portion of the parent’s resources are available to the sub-organization.
+The workspace use the resources of the organization that is allocated by the system administrator. The resources for sub-organizations are taken from the parent organization. Administrators control which resources of the parent organization are available to the sub-organization.
 
 [id="managing-limits"]
 == Managing Limits
 
-*Action restricted to*: Eclipse Che system administrator and admins of the organization.
+The system configuration defines the default limits. The administrator of the organization manages only the limits of its sub-organizations. No resource limits applies to the organization by default.  The following are the limits defined by the system adminstrator:
+
+* *Workspace Cap*: The maximum number of workspaces that exists in the organization. +
+* *Running Workspace Cap*: The maximum number of workspaces that run simultaneously in the organization. +
+* *Workspace RAM Cap*: The maximum amount of RAM that a workspace uses in GB.
 
-The organization default caps are taken from the system configuration. The admin of the organization can manage only the limits of it’s sub-organizations. By default, there are no resource limits applied to the organization so all members can benefit from all the allocated resources. If an organization admin wishes to set limits they have three options:
+[NOTE]
```

Comment:
I'd move this admonition to the beginning of this section; there's no need to read this if you're not a system administrator of the administrator of an organization.